packages/block-library/src/pattern/index.php

```diff
@@ -43,8 +43,8 @@ function render_block_core_pattern( $attributes ) {
 	$pattern = $registry->get_registered( $slug );
 	$content = _inject_theme_attribute_in_block_template_content( $pattern['content'] );
 
-	$gutenberg_experiments = get_option( 'gutenberg-experiments' );
-	if ( $gutenberg_experiments && ! empty( $gutenberg_experiments['gutenberg-block-hooks'] ) ) {
+	// This can be removed when the minimum supported WordPress is >= 6.4.
+	if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
```

Comment:
It's very likely that we will have to refactor this code once the final solution is fleshed out in https://github.com/WordPress/wordpress-develop/pull/5158.